.eslintrc

```diff
@@ -11,6 +11,10 @@
         "no-unused-vars": ["error", { "vars": "all", "args": "none", "ignoreRestSiblings": true }],
         "no-constant-condition": ["error", { "checkLoops": false }],
         "no-var": "error",
-        "no-useless-constructor": "error"
+        "no-useless-constructor": "error",
+        "prefer-arrow-callback": ["error", { "allowNamedFunctions": false, "allowUnboundThis": true }],
```

Comment:
These are the default options. Can't we just omit them in that case?

Reply:
I guess that we could remove the options yeah.
I just copy-pasted them from air-bnb style config
I guess that it's more explicit and less prone to errors when updating in case eslint changes defaults.

But I can remove them if you prefer that :)

Reply:
Very subjective I guess, but personally I would read such things as deviations from the defaults. :)